docs/src/assertions.md

```diff
@@ -131,27 +131,27 @@ Assert.True(checked);
 ## JS expression
 
 ```js
```

Comment:
Should we follow up with removing this file entirely from JavaScript docs? Same goes for other languages that implement web-first assertions?

Reply:
Yeah, it is already hidden-ish, but yes.